src/ExcelFiles.jl

```diff
@@ -27,6 +27,11 @@ function fileio_load(f::FileIO.File{FileIO.format"Excel"}, range; keywords...)
     return ExcelFile(f.filename, range, keywords)
 end
 
+function fileio_save(f::FileIO.File{FileIO.format"Excel"}, data)
```

Comment:
Could we add a keyword argument for the sheetname? With some sensible default, maybe say ``"Sheet1"``?